h/accounts/models.py

```diff
@@ -77,6 +77,8 @@ class User(Base):
                       server_default=sa.sql.expression.false(),
                       nullable=False)
 
+    admin = sa.Column(sa.BOOLEAN, default=False, nullable=False)
```

Comment:
To be clear, this *has* to have a `server_default` if it's not nullable, otherwise we'll break user registration in prod as soon as we run the migration for stage (they use the same database).